src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -354,7 +355,14 @@ export default React.memo(function GamePage(): JSX.Element | null {
                     }}
                   >
                     <b>{t('info.syncsaves')}:</b>{' '}
-                    {t('cloud_save_unsupported', 'Unsupported')}
+                    <FontAwesomeIcon
+                      className="helpIcon"
+                      icon={faCircleInfo}
+                      title={t(
+                        'cloud_save_unsupported',
```

Comment:
```suggestion
                        'help.cloud_save_unsupported',
```

Reply:
Ok i tested it and i found more problems.
This should fix it:
```typescript
                    {t('cloud_save_unsupported', 'Unsupported')}
                    <FontAwesomeIcon
                      className="helpIcon"
                      icon={faCircleInfo}
                      title={t(
                        'help.cloud_save_unsupported',
                        'This game does not support cloud saves. This information is provided by the game developers. Some games do implement their own cloud save system'
                      )}
                    />
```

Reply:
Thanks!